completions/completions_v2.py

```diff
@@ -94,13 +76,14 @@ def on_selection_modified(self, view):
         self.on_any_event(view)
     def on_activated(self, view):
         self.on_any_event(view)
-        if view.window():
-            view.window().status_message("TabNine")
+        view.set_status("tabnine-status", "TabNine")
```

Comment:
What this does?

Reply:
sets the status bar item to "TabNine" 
<img width="468" alt="Screen Shot 2020-10-18 at 12 08 23" src="https://user-images.githubusercontent.com/60742964/96363087-b9eca080-113a-11eb-8e1a-65bf3be57bbd.png">